storage/store.go

```diff
@@ -647,12 +647,14 @@ func (s *Store) MaybeGossipConfigs() error {
 		if _, err := rng.getLeaseForGossip(s.Context(nil)); err != nil {
```

Comment:
we ignore the first parameter here, which will cause non-lease holders to gossip. Why?